tasks/settings.yml

```diff
@@ -65,6 +58,27 @@
     group: "{{ jenkins_process_group }}"
     mode: 0775
 
+- name: Ensure jenkins_tmpdir {{ jenkins_tmpdir }} exists.
+  file:
+    path: "{{ jenkins_tmpdir }}"
+    state: directory
+    owner: jenkins
+    group: jenkins
+    mode: u+rwx
+    follow: true
+  when: jenkins_tmpdir | length > 0
+
+- name: "Force systemd to be reload before atempting to restart jenkins"
```

Comment:
May be remove quotes here ?
Should'nt it says "to be reloaded" ?

Reply:
fixed